src/app/home/pages/PlanDebugPage/components/TreeActionsDropdown.tsx

```diff
@@ -39,7 +39,7 @@ const TreeActionsDropdown: React.FunctionComponent<ITreeActionsDropdownProps> =
     clipboard.removeChild(el);
     dispatch(
       alertSuccess(
-        `Command copied to clipboard. Run 'oc get' on 
+        `Command copied to clipboard. Run 'oc describe' on the 
```

Comment:
I think this got lost in the rebase, all `oc get` copy to clipboards got swapped out for `oc describe` commands.